pisek/jobs/parts/solution.py

```diff
@@ -130,31 +130,41 @@ def _update(self):
                 subtask.cancel()
 
     def _get_status(self) -> str:
-        msg = f"Testing {self.solution_label} "
+        msg = f"Testing {self.solution_label}"
         if self.state == State.canceled:
             return self._job_bar(msg)
-        msg = pad(msg, MSG_LEN)
 
-        INT_PLACES = len(str(self._env.config.total_points))
-        DEC_PLACES = 2
-        if self.solution_points is None:
-            points = "?" + "." + "?" * DEC_PLACES
+        points_places = len(str(self._env.config.total_points)) + POINTS_DEC_PLACES + 2
+        points = f"{format_points(self.solution_points)}p"
+
+        max_time = max((s.slowest_time for s in self.subtasks), default=0)
+
+        verbosity = self._env.verbosity if self.state.finished() else 0
+        subtasks_res = [sub.status(self.subtasks, verbosity) for sub in self.subtasks]
+
+        if verbosity <= 0:
+            points = pad_left(points, points_places + 1)
```

Comment:
Why add 1 here? Wouldn't it be clearer if the space was added in the format string a line below?